### DISCUSSION TITLE
Suggestion for improvement for the font settings in the widgets

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#743] [Ideas]
Today I noticed another unpleasant thing with the widgets, I think I had already mentioned this some time ago. It's about the font settings in the widgets. I use local fonts and set this in the widgets and save it (image 1).

![font_settings_1](https://github.com/user-attachments/assets/b78df4a2-fcfe-4927-ba45-2df0ef6b390e)

Everything works fine and the fonts are displayed on the website as saved. But when I then edit the widget again, the set fonts are no longer visible, only the font weight is still visible as set (image 2).  

![font_settings_2](https://github.com/user-attachments/assets/1089a411-aa21-4a65-b1ed-483db77cc18e)

I find this very inconvenient because you always have to check whether you have set the right fonts or whether they are still set. If you live in the EU, you have to be very careful because of the GDPR that you don't accidentally load a Google font. It would be nice if it was like in the Astroid Manager under the typography settings, where you can always see which font settings are currently being used.

Best Regards

Translated with DeepL.com (free version)

**Comment by Banuk1616** [comment]
Am I the only one who has this problem? Is the problem perhaps due to a mistake I am making somewhere? I'm now so annoyed by having to check all the widgets I use and reassign the fonts (because you can't see which font is used in the widget) if a Google font has been loaded somewhere unintentionally that I'm considering switching back to third-party extensions. That's a pity, because I think the system with the widget integrated in the Astroid framework is simply brilliant.

**Comment by sonvnn** [comment]
Thank you! I will check it and response you asap :)

**Comment by sonvnn** [comment]
@Banuk1616 I fixed this issue at https://github.com/templaza/astroid-framework/commit/41bb2fabf295354edae9af753e101d107117c341 You can download v3-branch to check it.

**Comment by Banuk1616** [comment]
@sonny
I just tested v3 Brunch briefly with the slideshow widget. When reopening the widget after saving, the saved fonts are now displayed in the widget backend. It works great now. Thank you very much!
Best regards
